modules/core/src/main/scala/fs2/kafka/RecordDeserializer.scala

```diff
@@ -19,13 +19,34 @@ sealed abstract class RecordDeserializer[F[_], A] {
 
   def forValue: F[Deserializer[F, A]]
 
+  /**
+    * Returns a new [[RecordDeserializer]] instance applying the specified mapping functions to key and value deserializers
+    */
+  final def bimap[B](
+    fKey: Deserializer[F, A] => Deserializer[F, B],
+    fValue: Deserializer[F, A] => Deserializer[F, B]
+  )(implicit F: Functor[F]): RecordDeserializer[F, B] =
```

Comment:
Good point, I don't have a strong opinion but I would go with `map` as you suggested. I'd like to have both actually but then it may be something useless to maintain.  
This also allows having the `Functor` instance

Reply:
Good points - I don't think `bimap` would be a big maintenance burden but I'd rather not add it unless there's an actual use case.

Feel free to make the change and then I'll merge this.

Reply:
Sorry @bplommer, one more note about naming. 
`.map` ideally implies this signature
```scala
 def map[B](f: A => B): RecordSerializer[F, B]
```

But we have this
```scala
 def map[B](f: Serializer[F, A] => Serializer[F, B]): RecordSerializer[F, B]
```

Do you think `map` is fine or should we find a better name ( like `mapSerializer` | `mapDeserializer` ? )?

Reply:
Good point, I don't think we should call this `map`. What do you think of `transform`?

Reply:
I like it! Updating the PR 👍